go/service/chat_local.go

```diff
@@ -1543,3 +1565,9 @@ func (h *chatLocalHandler) FindConversationsLocal(ctx context.Context,
 	res.RateLimits = utils.AggRateLimits(res.RateLimits)
 	return res, nil
 }
+
+// perfLog logs function call durations in ms.
+//    example usage: defer h.perfLog("MyFunc", time.Now())
+func (h *chatLocalHandler) perfLog(name string, start time.Time) {
+	h.G().Log.Debug("ChatPerf %s: %d", name, time.Since(start)/time.Millisecond)
```

Comment:
Could do something like:

```go
func (h *chatLocalHandler) perfLog(name string) func() {
    start := time.Now()
    return func() {
       h.G().Log.Debug("ChatPerf %s: %d", name, time.Since(start)/time.Millisecond)
    }
}

...

defer h.perfLog("mike")()
```

Up to you though.